Build/Strategic/Map_Screen_Helicopter.cc

```diff
@@ -436,21 +383,14 @@ BOOLEAN IsHelicopterPilotAvailable( void )
 	if (iHelicopterVehicleId == -1) return FALSE;
 
 	// owe any money to skyrider?
-	if( gMercProfiles[ SKYRIDER ].iBalance < 0 )
-	{
-		return( FALSE );
-	}
+	if( gMercProfiles[ SKYRIDER ].iBalance < 0 ) return( FALSE );
 
 	// Drassen too disloyal to wanna help player?
-	if ( CheckFact( FACT_LOYALTY_LOW, SKYRIDER ) )
-	{
-		return ( FALSE );
-	}
+	if ( CheckFact( FACT_LOYALTY_LOW, SKYRIDER ) ) return ( FALSE );
```

Comment:
You're right and it looks readable to me — especially if I close my eyes a bit and ignore the odd spacing (imo).